ncm-spma/src/main/pan/components/spma/functions-rpm.pan

```diff
@@ -0,0 +1,572 @@
+# ${license-info}                                                                                                                                                                                                                            
```

Comment:
also is the the `yumng` or old `yum`? (i'm confused with the `-rpm` and the `-yum`, and the lack of `-yumng`)